data/definitions/variables.yml

```
@@ -298,6 +298,24 @@ quest_points:
   id: 101
   type: varp
   persist: true
+shield_of_arrav_reldo:
+  id: 145
+  type: varp
+  persist: true
+  format: map
+  values:
+    unstarted: 0
+    started: 1
+    completed: 7
+shield_of_arrav_charlie:
+  id: 146
+  type: varp
+  persist: true
+  format: map
+  values:
```

```suggestion
  format: map
  default: unstarted
  values:
```
You need to add a default if using `.getVar<String>("shield_of_array_charlie")` otherwise it'll throw an exception when a player doesn't have that variable set.

Alternatively use 
`if (player.getVar<String>("shield_of_array_charlie", "unstarted") == "unstarted") {` if more flexible defaults values are needed